dev/devicelab/bin/tasks/android_java11_dependency_smoke_tests.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:file/local.dart';
+import 'package:flutter_devicelab/framework/apk_utils.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+
+List<VersionTuple> versionTuples = <VersionTuple>[
```

Comment:
Consider linking to the documentation for where we define the lowest supported versions and where we know that these versions are java 11 compatible.

Reply:
Added a link to the file where we define range, and added a link to compatibility matrix explaining the choice of Java 11